volume_manager/Utils.cpp

```diff
@@ -239,10 +239,10 @@ status_t ForkExecvp(const std::vector<std::string>& args, security_context_t con
         }
     }
 
-    if (setexeccon(context)) {
-        LOG(ERROR) << "Failed to setexeccon";
-        abort();
-    }
+    // if (setexeccon(context)) {
```

Comment:
Please just remove the code. There is no use in commented-out code parts, we have Git, and in a few months to years nobody will know why this is commented out :)

Reply:
Done